app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsPresenter.kt

```diff
@@ -137,6 +142,19 @@ class SkillDetailsPresenter(skillDetailsFragment: SkillDetailsFragment) : ISkill
         }
     }
 
+    override fun reportSendError(throwable: Throwable) {
+        Timber.d(throwable)
+        skillDetailsView?.updateSkillReportStatus("Coudn't report the skill")
```

Comment:
Use string resource.

Reply:
@arundhati24 How can i get the context, it's presenter class.

Reply:
You can check SkillListingPresenter for reference.